src/documentation/0009-node-environment-variables/index.md

```diff
@@ -1,12 +1,30 @@
 ---
 title: How to read environment variables from Node.js
 description: 'Learn how to read and make use of environment variables in a Node.js program'
-authors: flaviocopes, MylesBorins, fhemberger, LaRuaNa, ahmadawais
+authors: flaviocopes, MylesBorins, fhemberger, LaRuaNa, ahmadawais, aymen94
 section: Getting Started
 ---
 
 The `process` core module of Node.js provides the `env` property which hosts all the environment variables that were set at the moment the process was started.
 
+## We have two way to set variables:
+- using the command line
```

Comment:
Could `the` word be omitted to align w. L14 header markdown? (or add `the` at L11, but prefer w/o `the` personally. Could agree w. either to move forward)

```suggestion
- using command line
```